betterproto/compile/importing.py

```diff
@@ -86,7 +86,7 @@ def reference_absolute(imports, py_package, py_type):
     string_import = ".".join(py_package)
     string_alias = safe_snake_case(string_import)
     imports.add(f"import {string_import} as {string_alias}")
-    return f"{string_alias}.{py_type}"
+    return f'"{string_alias}.{py_type}"'
```

Comment:
what are the extra quotes for?

Reply:
These make the typehints come out as

```python
class Example(betterproto.Message):
   foo: "__some_package__Message" = ...
```

It creates a forward reference that will allow referencing objects that are not imported yet.

See: https://www.python.org/dev/peps/pep-0484/#forward-references